package.json

```diff
@@ -36,8 +36,10 @@
     "eslint-config-prettier": "^2.9.0",
     "expect.js": "^0.3.1",
     "husky": "^0.14.3",
+    "js-yaml": "^3.12.0",
```

Comment:
Sorry I should have been more specific before. Only the `mock-fs` needs to be in devDeps. `js-yaml` would need to be in normal `dependencies` as it's being used in the main library file.

Reply:
oh duh...sorry wasn't thinking. updated.